app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -135,15 +135,19 @@
 }
 
 #downloadPurchaseButton {
-  float: right;
   background-color: #5cb85c;
-  margin-top: 10px;
 }
 
 #disapproveButton {
   background-color: #d9534f;
-  float: left;
-  margin-top: 10px;
+}
+
+#downloadButton {
```

Comment:
Avoid using id selectors<br>Selector `downloadButton` should be written in lowercase with hyphens